### PR TITLE
Basic brand identity

### LINKED ISSUE #25: Implement basic brand identity

### PR DESCRIPTION
**Pull request by mudrila**
## Changes

- Add basic brand identity with logos, colors, fonts and metadata
- Adjust theme creation and introduce "light theme"
- Minor dependencies / engines versions update

## Addressed Issues

- Closes #25

### CODE REVIEW

**Comment by Artemh59** [comment]
I just need to see what has changed, right?)

**Comment by mudrila** [comment]
> I just need to see what has changed, right?)

@Artemh59 
Yessir - but also if you have any comments regarding code changes (you feel like some stuff will be better done differently) - feel free to comment / request changes / or approve if everything looks good :)

**Comment by Artemh59** [comment]
Only one thing)

File: package.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
 {
-  "name": "fin-plan",
+  "name": "yfp",
   "author": {
     "name": "Kyrylo Klymenko & Artem Harchenko",
```
Comment (Artemh59) [inline package.json:4]: Kharchenko

**Comment by Artemh59** [comment]
Otherwise I can only study :)

File: package.json (hunk 1 above)
Comment (mudrila) [reply package.json]: Fixed sir - thank you for noticing!

**Pull request merged**